README.md

```diff
@@ -1 +1,23 @@
 # ansible-python-monasca-events
+
+Installs the [python-monasca-events](https://github.com/hpcloud-mon/python-monasca-events) part of the [Monasca](https://wiki.openstack.org/wiki/Monasca) project.
+
+The role install the cli into a virtualenv on the box.
+
+## Requirements
+virtualenv must be installed on the system. 
+All dependencies {babel, argparse, iso8601, pbr, PrettyTable, PyYAML, requests} should be installed
+
```

Comment:
If we're already installing the dependencies as part of the CLI install I don't think that we need to mention them there.